buildtest/cli/help.py

```diff
@@ -310,9 +310,14 @@ def print_inspect_help():
         "Display all test names, ids, and corresponding buildspec file",
     )
     table.add_row("buildtest inspect list -t", "Show output in terse format")
+    table.add_row("buildtest inspect list --pager", "Paginate output of inspect list")
     table.add_row(
         "buildtest inspect name hello", "Display last run for test name 'hello'"
     )
+    table.add_row(
+        "buildtest inspect name hello --pager",
+        "Paginate output of last run for test name 'hello'",
+    )
```

Comment:
yeah i think we dont need to document every single `--pager` option for every sub-command. I think its sufficient to just have one like `buildtest inspect list --pager`. Otherwise we will end up having too many outputs.